guillotina_oauth/oauth.py

```diff
@@ -172,7 +172,7 @@ async def service_token(self):
             now = timegm(datetime.utcnow().utctimetuple())
             if (self._service_token["exp"] - 60) > now:
                 return self._service_token["service_token"]
-        return await self.refresh_service_token()
+        return self.refresh_service_token()
```

Comment:
Did you remove this await on accident?

Reply:
No, I removed it because the test was failing:

```
    @property
    async def service_token(self):
        if self._service_token:
            now = timegm(datetime.utcnow().utctimetuple())
            if (self._service_token["exp"] - 60) > now:
                return self._service_token["service_token"]
>       return await self.refresh_service_token()
E       TypeError: object str can't be used in 'await' expression
```

Reply:
This doesn't seem right, refresh_service_token is an async method, this logic is going to fail without that await.

Reply:
Yeah, you are right. What was wrong was the mock done in the test. Hopefully it's fixed now.